index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="initial-scale=1, width=device-width"/>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
     <title>Getting Started with PASS</title>
-    <link rel="stylesheet" href="/src/style.css"/>
+    <link rel="stylesheet" href="/src/style.css" />
     <link rel="icon" type="image/x-icon" href="src/assets/pass-logo.png">
```

Comment:
I've just noticed, but the link in Line 8 is missing a self-closing tag `/>` when compared to the lines above it.